content/hi/cloud-native-security.md

```diff
@@ -0,0 +1,18 @@
+---
+title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["सुरक्षा"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव अनुप्रयोगों में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर निर्माण तक संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है। क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [DevSecOps](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
+
+## समस्या जिस का यह समाधान करता है
+
+पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। [क्लाउड नेटिव ऐप](/cloud-native-apps/) (अनुप्रयोग) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और बुनियादी ढांचे में तेज़ बदलाव के अधीन होते हैं। कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, मेज़बान-आधारित भेद्यता अवलोकन और अंतिम मिनट की सुरक्षा समीक्षा क्लाउड नेटिव अनुप्रयोगों के साथ [स्केल](/scalability/) नहीं की जा सकती हैं। 
+
+## यह कैसे मदद करता है
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा प्रतिरूप से स्थानांतरित करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है। हस्तचालित परीक्षण और जांच को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया गया है। त्वरित कोड रिलीज़ पाइपलाइन के उन उपकरणों के साथ एकीकृत हैं जो संकलित होने से पहले भेद्यता के लिए कोड की समीक्षा करते हैं। ओपन सोर्स लाइब्रेरी विश्वसनीय स्रोतों से ली जाती हैं और अरक्षितता के लिए उनकी निगरानी भी की जाती है। बदलाव को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा प्रतिरूप अक्सर अद्यतन किए गए असुरक्षित घटकों द्वारा इसे ग्रहण करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदला जा रहा है।
```

Comment:
rephrase this part "करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है" cause it seems out of place. something like -
"...पारंपरिक सुरक्षा प्रतिरूप से स्थानांतरित करती है वहाँ जहां अनुप्रयोगों की सुरक्षा विमोचन (रिलीज़) चक्र के हर कदम में सुरक्षा शामिल होती है।"

Reply:
How about replacing the whole sentence with "क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है। यह पारंपरिक सुरक्षा प्रतिरूप को स्थानांतरित करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है।" ?